activerecord/lib/active_record/fixtures.rb

```diff
@@ -456,6 +456,31 @@ class FixtureClassNotFound < ActiveRecord::ActiveRecordError # :nodoc:
   # In the above example, 'base' will be ignored when creating fixtures.
   # This can be used for common attributes inheriting.
   #
+  # == Composite Primary Key Fixtures
+  #
+  # If a model has a composite primary key, both composite key columns should
+  # be specified in its fixtures:
+  #
+  #   # app/models/order.rb
+  #   class Order < ApplicationRecord
+  #     self.primary_key = [:shop_id, :number]
+  #   end
+  #
+  #   # orders.yml
+  #   shop_id: <%= ActiveRecord::FixtureSet.identify(:book_store) %>
+  #   number: 1056
+  #
+  # When using an id column, it may be omitted just like in normal fixtures:
```

Comment:
To be fair full CPK can be omited and the value will be auto-generated. It may not always make sense but it will be unique

Reply:
I started to doubt the usefulness of this first section, so I merged it with the second and dropped the first order example. I also added a relationship section that shows how linking is currently possible. Let me know what you think.